### PR TITLE
Added support for new event type (v3)

### LINKED ISSUE #72: Recent versions of Snort unified2 not supported.
Hello,

I am trying to convert snort (=2.9.11) log to json file but getting error on python 2.7.12 with idstools (=0.6.3)

the code:

`
#!/usr/bin/python
from idstools import unified2

reader = unified2.SpoolRecordReader("/var/log/snort",
    "snort.log", follow=True)
for record in reader:
    if isinstance(record, unified2.Event):
        print("Event:")
    elif isinstance(record, unified2.Packet):
        print("Packet:")
    elif isinstance(record, unified2.ExtraData):
        print("Extra-Data:")
    print(record)
`

error:
**
Traceback (most recent call last):
  File "/home/zero/Desktop/log_2_json/for_snort.py", line 6, in <module>
    for record in reader:
  File "/home/zero/.local/lib/python2.7/site-packages/idstools/unified2.py", line 846, in iter_next
    record = self.next()
  File "/home/zero/.local/lib/python2.7/site-packages/idstools/unified2.py", line 817, in next
    record = self.reader.next()
  File "/home/zero/.local/lib/python2.7/site-packages/idstools/unified2.py", line 610, in next
    return read_record(self.fileobj)
  File "/home/zero/.local/lib/python2.7/site-packages/idstools/unified2.py", line 571, in read_record
    raise UnknownRecordType(rtype)
idstools.unified2.UnknownRecordType: Unknown record type: 3569595041
**

my snort log directory have files:
**
ls /var/log/snort
archived_logs  snort.log.1547718241  snort.log.1547725234
**
help me man.

### PR DESCRIPTION
**Pull request by ArthanJans**
Fixes #72

### CODE REVIEW

File: idstools/unified2.py [hunk 1]
```diff
@@ -813,23 +854,16 @@ def next(self):
...
+        while True:
+            try:
+                record = self.reader.next()
+            except EOFError:
+                return
+            if record:
+                return record
+            else:
+                if not self.open_next():
+                    return
```
Comment (jasonish) [inline idstools/unified2.py]: Is this required for the new record type? If not, please break it out into its own commit please.

**Comment by jasonish** [comment]
Do you have a sample of this log file we can include for tests?

**Comment by ArthanJans** [comment]
[u2logs.zip](https://github.com/jasonish/py-idstools/files/3276982/u2logs.zip)
Here are some sample log files for testing. Be warned that I have not checked that they are exhaustive and there may be some things that remain untested

**Comment by csbflyer** [comment]
Is there a plan to merge this patch? Does it fix parsing latest snort event types? I don't see them listed here: https://idstools.readthedocs.io/en/latest/_modules/idstools/unified2.html
